docs/docs/external/up-quick-start.md

```diff
@@ -0,0 +1,23 @@
+# Quick start
+
+:::info This guide is meant to be included in the up.aztec.network site and not in the main documentation.
```

Comment:
We'll also need the curl command to install and start the sandbox.

Reply:
Agree, that'll be separate in the up page

Reply:
Copy-paste :-/